### PR TITLE
Minor fixes & integration of Payment

### PR DESCRIPTION
**Pull request by aayushjain-code**

### CODE REVIEW

**Review by nayyyhaa — APPROVED** [review]
Nice work! Found nothing much to suggest. You can merge the changes.

File: src/Components/Navbar/Navbar.js [hunk 1]
```diff
@@ -39,15 +39,15 @@ const Navbar = () => {
           {" "}
           <i className="fab fa-canadian-maple-leaf"></i> FreshStore
         </Link>
-        <form action="" class="search-form">
+        {/* <form action="" class="search-form">
```
Comment (MuskanChhatrasal) [inline src/Components/Navbar/Navbar.js:42]: Remove commented code

File: src/Pages/Products/products.css [hunk 2]
```diff
@@ -74,7 +87,10 @@
 	right: 1rem;
 }
 
-
-.button{
-	width: 40%;
+.sale-tag{
+    padding: 0.05rem 1.2rem !important;
```
Comment (MuskanChhatrasal) [inline src/Pages/Products/products.css:91]: Try to avoid important keyword

File: src/Utils/root.css [hunk 3]
```diff
@@ -32,4 +32,5 @@ body.active {
 
 .btn-success{
 	background-color: var(--primary);
+	color:white;
```
Comment (MuskanChhatrasal) [inline src/Utils/root.css:35]: Use colour which are predeclared in root folder

**Review by MuskanChhatrasal — APPROVED** [review]
Amazing and cleaner UI. Just a few suggestions remove all the commented code and avoid using important keyword.

File: src/Components/Navbar/Navbar.js (hunk 1 above)
Comment (aayushjain-code) [reply src/Components/Navbar/Navbar.js:42]: Sure

File: src/Pages/Products/products.css (hunk 2 above)
Comment (aayushjain-code) [reply src/Pages/Products/products.css:91]: okay

File: src/Utils/root.css (hunk 3 above)
Comment (aayushjain-code) [reply src/Utils/root.css:35]: will do that

**Pull request merged**